src/util/XMLUtils.cpp

```diff
@@ -13,20 +13,22 @@
  *  GNU General Public License for more details.
  *
  *  You should have received a copy of the GNU General Public License
- *  along with XBMC; see the file COPYING.  If not, write to
- *  the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
- *  http://www.gnu.org/copyleft/gpl.html
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
  *
  */
 
 #include "XMLUtils.h"
+#include "StringUtils.h"
+#ifdef TARGET_WINDOWS
+#include "PlatformDefs.h" //for strcasecmp
+#endif
 
 bool XMLUtils::GetHex(const TiXmlNode* pRootNode, const char* strTag, uint32_t& hexValue)
 {
   const TiXmlNode* pNode = pRootNode->FirstChild(strTag );
   if (!pNode || !pNode->FirstChild()) return false;
-  sscanf(pNode->FirstChild()->Value(), "%x", (uint32_t*) &hexValue );
-  return true;
+  return sscanf(pNode->FirstChild()->Value(), "%x", (uint32_t*)&hexValue) == 1;
```

Comment:
The cast comes from the code in kodi code. We should keep those in sync.
I agree that the sync of XMLUtils should be in its own commit.